tests/acceptance/test_api.py

```diff
@@ -7,6 +7,7 @@ class ApiTokensTest(AcceptanceTestCase):
     def setUp(self):
         super(ApiTokensTest, self).setUp()
         self.user = self.create_user(email="foo@example.com", name="User Name")
+        self.org = self.create_organization(owner=self.user, name="foo")
```

Comment:
Previously, this test was passing even though there was no organization so I am assuming the org details endpoint just errored out gracefully and allowed the user to move on. However, now the organization context continually loads if it is detailed and does not have projects and teams so this organization object is required.

Reply:
woops I was mistaken adding a fix now